src/components/Books.js

```diff
@@ -1,21 +1,57 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
+import '../styles/Books.css';
```

Comment:
- [ ] The books section is not styled properly. When adding books, the new book is added in a new column which breaks the styling and the page.
![image](https://user-images.githubusercontent.com/33767951/167233023-dd9765e6-53f8-44c0-a66b-7e1917ec3a5f.png)